src/projects/components/parts/project-form/ProjectForm.tsx

```diff
@@ -28,6 +29,12 @@ export function ProjectForm({ project, onProjectChange }: ProjectFormProps) {
         },
     });
     const onSave = (project: Project) => {
+        // TODO: ugly temporal. We probably should refactor this a bit more.
+        if (avoidSave) {
```

Comment:
Not amazed about this one. I didn't want to change a lot how the update uses this component.

But I think we should move up the ´executeSave´ into the edit component itself instead of having it here.